app/experimenter/nimbus-ui/src/components/SlugTextControl/index.tsx

```diff
@@ -0,0 +1,26 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from "react";
+import Form from "react-bootstrap/Form";
+import { slugify } from "./slugify";
+
+export type SlugTextControlProps = {
+  defaultValue?: string;
+  value?: any;
+};
+
+export const SlugTextControl = ({
+  value,
+  defaultValue,
+}: SlugTextControlProps) => (
+  <Form.Control
+    readOnly
+    data-testid="SlugTextControl"
+    value={slugify(value || defaultValue)}
+    type="text"
+  />
```

Comment:
Oh yeah interesting I've also heard people ask about this for the regular create form.  This is great after we land this we should also backport this to the regular form.

Reply:
Yeah, should be close to a one-liner plus tests. Just a small component that takes a value and shows a slugified preview